File: MPP_MusicFestival_Java/src/main/java/domain/Entity.java
package domain;

public class Entity<T> {

    private T Id;

    public Entity(T id) {
        Id = id;
    }

    public T getId() {
        return Id;
    }

    public void setId(T id) {
        Id = id;
    }

    @Override
    public String toString() {
        return Id.toString();

    }
}

File: MusicFestival/Domain/Entity.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MPP_MusicFestival.Domain
{
    class Entity<T>
    {
        public T Id { get; set; }
        public Entity(T id)
        {
            this.Id = id;
        }

        public override string ToString()
        {
            return this.Id.ToString();
        }
    }
}

